jekyll/_cci2/using-linuxvm.md

```diff
@@ -33,11 +28,29 @@ jobs:
 ```yaml
 version: 2.1
 jobs:
-  build:
+  my-job:
     machine: true
+    resource_class: large
 ```
 
-You can view the list of available images [here]({{ site.baseurl }}/2.0/configuration-reference/#available-linux-machine-images).
+You can view the list of available images [in the docs Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/#available-linux-machine-images), or on the [Developer Hub](https://circleci.com/developer/images?imageType=machine). If you are working on an installation of CircleCI server, you will notice in the example above the syntax is slightly different, and the available Linux images are managed by your system administrator.
+
+## Available LinuxVM resource classes
+{: #available-linuxvm-resource-classes } 
+
+{% include snippets/machine-resource-table.md %}
+
+## Pre-installed software
+{: #pre-installed-software }
+
+The most up to date list of pre-installed software can be found on the [image builder](https://raw.githubusercontent.com/circleci/image-builder/picard-vm-image/provision.sh) page. You can also visit the [Discuss](https://discuss.circleci.com/) page for more information.
```

Comment:
I wonder if it would be appropriate / better to link to a more specific place in Discuss, like with the tag (e.g. https://discuss.circleci.com/tag/linux, I think there's a linux-vm tag as well), or maybe more broadly the CircleCI images category? https://discuss.circleci.com/c/ecosystem/circleci-images/64

Reply:
Yes probably, I think I just don't quite know where. I'll go with https://discuss.circleci.com/tag/machine-images I think. Seems to catch recent updates. Let me know if you think I should use a different tag.